src/pages/Dashboard/MyAccount/index.tsx

```diff
@@ -35,19 +33,17 @@ const MyAccount: React.FC<IMyAccountProps> = ({
     | 'borrowBalanceCents'
     | 'borrowLimitCents'
   > = React.useMemo(() => {
-    const borrowBalanceCents = userTotalBorrowBalance.multipliedBy(100);
-    const { supplyBalanceCents, yearlyEarningsCents } = calculateYearlyEarningsForAssets({
-      assets,
-      borrowBalanceCents,
-      isXvsEnabled,
-    });
+    const { borrowBalanceCents, supplyBalanceCents, yearlyEarningsCents } =
+      calculateYearlyEarningsForAssets({
+        assets,
+        isXvsEnabled,
+      });
     const netApyPercentage =
       supplyBalanceCents &&
       yearlyEarningsCents &&
       calculateApy({ supplyBalanceCents, yearlyEarningsCents });
     const dailyEarningsCents =
       yearlyEarningsCents && +calculateDailyEarningsCents(yearlyEarningsCents).toFixed(0);
```

Comment:
I know this is out of scope of this PR, but given we're currently always going to be using `calculateYearlyEarningsForAssets` to calculate the value that's then passed to `calculateDailyEarningsCents` wouldn't it make more sense to pass `assets` to `calculateDailyEarningsCents` instead and call `calculateYearlyEarningsForAssets` from within it?

Reply:
We could. I like how this clearly differentiates calculation per asset and the summation of all the assets.

Reply:
You can still have that by having two functions, but when it comes to how they'll be used in the codebase, given the current designs, we'll always use `calculateDailyEarningsCents` with `calculateYearlyEarningsForAssets` so this way we'd only have to import one function every time instead of two.

Reply:
You only have to import on function currently, unless I am missing something

Reply:
What I mean is you have to import `calculateYearlyEarningsForAssets` and `calculateDailyEarningsCents`, but because we don't actually use the value returned by `calculateYearlyEarningsForAssets`, we only pass it to `calculateDailyEarningsCents`. So if `calculateDailyEarningsCents` used the assets to call `calculateDailyEarningsCents` from within it, we'd only have to import and call `calculateDailyEarningsCents`

Reply:
We use the value for return value from `calculateYearlyEarningsForAssets` in our Net APY calculation. I like `calculateDailyEarningsCents` as a separate function that does not call `calculateYearlyEarningsForAssets` because it has a single responsibility